org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintBackendService.java

```diff
@@ -158,6 +163,41 @@ private static String getIdeName() {
     return ideName;
   }
 
+  /**
```

Comment:
I am not convinced of the value of the javadoc here. This is not really an API, and most infos can be guessed from method signature

Reply:
Yeah I agree, shrunk the comment to only contain a short summary. For me a short summary is necessary, as the method names differ so much, I would play cluedo the next time I encounter that after a long period trying to figure out why the first one contains `Description` and the other one `Details` 👍